### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by milesfrain**

### CODE REVIEW

File: 21-Hello-World/01-Prelude-ish/02-Prelude/08-Control-Flow--Functor-to-Monad/ReadMe.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ These will be covered at a slower and clearer pace in the upcoming files. This i
...
 | [Applicative](https://pursuit.purescript.org/packages/purescript-prelude/4.1.0/docs/Control.Applicative) | Liftable <hr> Parallel Computation | `pure :: forall a. a -> f a` |  | <ul><li>identity: `(pure (\x -> x) <*> v == v)`</li><li>composition: `pure (<<<) <*> f <*> g <*> h == f <*> (g <*> h)`</li><li>Homomorphism: `(pure f) <*> (pure x) == pure (f x)`</li><li>interchange: `u <*> (pure y) == (pure (_ $ y)) <*> u`</li></ul> | Put a value into a box <hr> Run code in parallel |
-| [Bind](https://pursuit.purescript.org/packages/purescript-prelude/4.1.0/docs/Control.Bind) | Sequential Computation | `bind :: forall m a b. m a -> (a -> m b) -> m b` | `>>=` <br> (Left 1)| Associativity: `(x >>= f) >>= g == x >>= (\x' -> f x' >>= g)` | Given an value of a box-like type, `m`, that contains a value, `a`, extract the `a` from `m`, and create a new `m` value that stores a new value, `b`. <br> Take `f a` and compute it via `bind`/`>>=` to produce a value, `a`. Then, use `a` to describe (but not run) a new computation, `m b`. When `m b` is computed (via a later `bind`/`>>=`), it will return `b`. |
```
Comment (milesfrain) [inline 21-Hello-World/01-Prelude-ish/02-Prelude/08-Control-Flow--Functor-to-Monad/ReadMe.md:40]: "Given **an** value" converted to "Given **a** value"

**Comment by JordanMartinez** [comment]
Wow... can't believe I messed up that header.

This is technically a breaking change because the header is changing.

**Comment by JordanMartinez** [comment]
Mind if I don't merge this until after the next PS release comes out? Then I can merge this after merging #451

**Comment by milesfrain** [comment]
Sure. No rush.

File: 21-Hello-World/01-Prelude-ish/02-Prelude/08-Control-Flow--Functor-to-Monad/ReadMe.md (hunk 1 above)
Comment (JordanMartinez) [reply 21-Hello-World/01-Prelude-ish/02-Prelude/08-Control-Flow--Functor-to-Monad/ReadMe.md:40]: Yeah, the diff didn't really show that (not your fault). Took a while to figure out what was changed.

**Comment by JordanMartinez** [comment]
I've changed the base branch to `development` and merged locally on my side. There weren't any issues, so I'll merge it here. Thanks for your work!

**Comment by JordanMartinez** [comment]
Hmm... seems like I should have merged it into the v0.14-syntax branch. Well, I did that locally and pushed to it in #451

**Pull request merged**